test/unit/app-child-manager.spec.js

```diff
@@ -178,6 +178,28 @@ describe('App Manager', function() {
     it('should be called three times', function() {
       expect(this.myApp.addChildApp.callCount === 3).to.be.true;
     });
+
+    it('should assign the childApp name to the passed in appName', function(){
+      expect(this.myApp.addChildApp('cA4', Marionette.Toolkit.App)).to.have.property('_name', 'cA4');
+    });
+
+  });
+
+  describe('getName', function(){
```

Comment:
so all of the other describes are written the same.  So for now we'll keep the consistency, but should open a new issue if the describes should be more verbose.